src/main/resources/fabric.mod.json

```
@@ -29,7 +29,7 @@
   "depends": {
     "fabricloader": ">=0.7.4",
```

make sure you bump this

```suggestion
    "fabricloader": ">=0.8.8",
```